src/test/java/org/fcrepo/importexport/common/BagDeserializerTest.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.importexport.common;
+
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Objects;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Test that zip, tar, and tar+gz extraction works as expected
+ *
+ * @author mikejritter
+ * @since 2020-02-13
+ */
+public class BagDeserializerTest {
+
+    public static final String BEYONDTHEREPOSITORY_JSON = "src/main/resources/profiles/beyondtherepository.json";
+    public static final String BAG_INFO_TXT = "bag-info.txt";
+    public static final String DATA_DIR = "data";
+    private final String expectedDir = "bag";
+    private final String group = "compress";
+    private Path target;
+
+    @Before
+    public void setup() throws URISyntaxException {
+        final URL sample = this.getClass().getClassLoader().getResource("sample");
+        target = Paths.get(Objects.requireNonNull(sample).toURI());
+        Assert.assertNotNull(target);
```

Comment:
A very minor improvement:  I would suggest statically importing the assert methods to reduce the verbosity: 

Assert.assertNotNull becomes assertNotNull 

and add: import static org.junit.Assert.assertNotNull.

Same with assertTrue() and fail().